layouts/_default/list.html

```diff
@@ -12,9 +12,21 @@
     {{ end }}
 {{ end }}
 
+{{ define "head-block" }}
```

Comment:
The tooltips were an attempt to mimic what the current website shows below each button. I've removed for now, if we want to pull that text back in, I will adjust.